Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathSchemaBuilder.php

```diff
@@ -22,8 +22,7 @@ public function __construct(
     public function buildSchema(AbstractSchemaManager $schemaManager): Schema
     {
         $schema = DbalSchemaFactory::createSchemaWithTables($schemaManager, [
-            $this->createUriTable(),
-            $this->createLiveContentStreamsTable()
```

Comment:
Also a more general question:
Wouldn't this be also responsible for now removing the deprecated table?

Reply:
"Create" is a bit misleading here – it creates the schema for the table. There is no way to _remove_ a schema here.
That's currently done in the projection itself (see https://github.com/neos/neos-development-collection/pull/5149/files#diff-2e3df51a3340b51e08cf9d2ba5aade329a8c0e073354c1d5a72e42c3aaf5a127R112)
But I'm with @kitsunet on this one: Let's leave those dangling for now. For 9.0 -> 9.1 ect we'll have to think of a way to migrate those things once